setup.py

```diff
@@ -16,7 +16,7 @@
 
 extras_require = {
     # win32 APIs if on Windows (required for npipe support)
-    ':sys_platform == "win32"': 'pywin32==227',
+    ':sys_platform == "win32"': 'pywin32>=227',
```

Comment:
```suggestion
    ':sys_platform == "win32"': 'pywin32>=301',
```
maybe? to avoid https://github.com/advisories/GHSA-hwfp-hg2m-9vr2

Reply:
Sure. I just updated that.